File: sprint4/By languages/kotlin/lesson1code3.kt
val fruits = listOf("яблоко", "яблоко", "груша", "яблоко", "слива")
val fruitCount = mutableMapOf<String, Int>()  // создаём пустой ассоциативный массив
for (fruit in fruits) {
    if (!fruitCount.containsKey(fruit)) {     // если в отображении нет такого ключа
        fruitCount[fruit] = 0                 // заводим счетчик с таким ключом
    }
    fruitCount[fruit] = fruitCount[fruit]!! + 1 // увеличиваем счетчик
}

File: sprint4/By languages/swift/lesson1code3.swift
var fruits = ["яблоко", "яблоко", "груша", "яблоко", "слива"]
var fruitCount: [String: Int] = [:]

for fruit in fruits {
    if fruitCount[fruit] == nil {
        fruitCount[fruit] = 0
    }
    fruitCount[fruit]! += 1
}
